assets/js/nexi-checkout-inline.js

```diff
@@ -23,18 +24,23 @@ jQuery( function ( $ ) {
          * Initialize the gateway
          */
         init() {
+            wcNexiCheckout.selectedGateway = $( 'input[name="payment_method"]:checked' ).val()
             $( document ).ready( () => {
                 // When an update_order_review happens, WC will replace the payment methods fragment, resulting in the payment method element being replaced. Therefore, we have to listen on the body element.
                 $( "body" ).on( "change", 'input[name="payment_method"]', ( e ) => {
+                    const changedGateway = $( 'input[name="payment_method"]:checked' ).val()
+                    if( changedGateway !== "dibs_easy" && selectedGateway !== "dibs_easy" ) {
+                        return
+                    }
+
                     e.preventDefault()
 
                     // Updated the internal reference in case it was replaced by a fragment.
                     wcNexiCheckout.paymentMethodEl = $( e.target )
                     wcNexiCheckout.blockUI()
 
                     // Do not cache the payment method element, as it will be replaced by a fragment.
-                    const selectedGateway = $( 'input[name="payment_method"]:checked' ).val()
-                    wcNexiCheckout.changeSelectedGateway( selectedGateway === "dibs_easy" )
+                    wcNexiCheckout.changeSelectedGateway( changedGateway === "dibs_easy" )
 
                     // In case the payment method change fails due to an AJAX error, we want to prevent WC from updating the chosen payment method. Instead, the chosen payment method should be set by the AJAX handler which only happens if the transition was successful.
                     wcNexiCheckout.unblockUI()
```

Comment:
Suggestion: I think the comments in this part are now a bit wrong, since some code was moved around. Just to make this more easily maintained i think we should look at that before merging